tests/integration/flag_check/runner.py

```diff
@@ -0,0 +1,170 @@
+    cleanup(memgraph)
+
+
+def test_init_data_file(memgraph_args: List[str]):
+    memgraph = start_memgraph(memgraph_args)
+    execute_flag_check(flag_checker_binary, ["MATCH (n) RETURN n"], 2, "user", "user")
+    cleanup(memgraph)
+
+
+def test_init_and_init_data_file(memgraph_args: List[str]):
+    memgraph = start_memgraph(memgraph_args)
+    execute_with_user(["MATCH (n) RETURN n"])
+    execute_without_user(["MATCH (n) RETURN n"], True, "Handshake with the server failed!", True)
+    execute_flag_check(flag_checker_binary, ["MATCH (n) RETURN n"], 2, "user", "user")
+    cleanup(memgraph)
+
+
+def execute_test(memgraph_binary: str) -> None:
+    storage_directory = tempfile.TemporaryDirectory()
+    memgraph_args = [memgraph_binary, "--data-directory", storage_directory.name]
+
+    # Start the memgraph binary
+    with open(os.path.join(os.getcwd(), "dummy_init_file.cypherl"), "w") as temp_file:
+        temp_file.write("CREATE USER admin IDENTIFIED BY 'admin';\n")
+        temp_file.write("CREATE USER user IDENTIFIED BY 'user';\n")
+
+    with open(os.path.join(os.getcwd(), "dummy_init_data_file.cypherl"), "w") as temp_file:
+        temp_file.write("CREATE (n:RANDOM) RETURN n;\n")
+        temp_file.write("CREATE (n:RANDOM {name:'1'}) RETURN n;\n")
+
+    # Run the test with all combinations of permissions
+    print("\033[1;36m~~ Starting env variable check test ~~\033[0m")
+    test_without_any_files(memgraph_args)
+    memgraph_args_with_init_file = memgraph_args + [
+        "--init-file",
+        os.path.join(os.getcwd(), "dummy_init_file.cypherl"),
+    ]
+    test_init_file(memgraph_args_with_init_file)
+    memgraph_args_with_init_data_file = memgraph_args + [
+        "--init-data-file",
+        os.path.join(os.getcwd(), "dummy_init_data_file.cypherl"),
+    ]
+
+    test_init_data_file(memgraph_args_with_init_data_file)
+    memgraph_args_with_init_file_and_init_data_file = memgraph_args + [
+        "--init-file",
+        os.path.join(os.getcwd(), "dummy_init_file.cypherl"),
+        "--init-data-file",
+        os.path.join(os.getcwd(), "dummy_init_data_file.cypherl"),
+    ]
+    test_init_and_init_data_file(memgraph_args_with_init_file_and_init_data_file)
+    print("\033[1;36m~~ Ended env variable check test ~~\033[0m")
+
+    os.remove(os.path.join(os.getcwd(), "dummy_init_data_file.cypherl"))
+    os.remove(os.path.join(os.getcwd(), "dummy_init_file.cypherl"))
+
+
+if __name__ == "__main__":
+    memgraph_binary = os.path.join(PROJECT_DIR, "build", "memgraph")
+    tester_binary = os.path.join(PROJECT_DIR, "build", "tests", "integration", "flag_check", "tester")
+    flag_checker_binary = os.path.join(PROJECT_DIR, "build", "tests", "integration", "flag_check", "flag_check")
+
+    parser = argparse.ArgumentParser()
+    parser.add_argument("--memgraph", default=memgraph_binary)
+    parser.add_argument("--tester", default=tester_binary)
+    parser.add_argument("--flag_checker", default=flag_checker_binary)
```

Comment:
Are these arguments passed to the tests? If not, then please either remove them or pass them to the tests.